pkgs/data/icons/bibata-cursors/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchFromGitHub, gksu, inkscape, stdenv, xcursorgen }:
+
+stdenv.mkDerivation rec {
+  name = "bibata-cursors-${version}";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "KaizIqbal";
+    repo = "Bibata_Cursor";
+    rev = "v${version}";
+    sha256 = "14gvpjp4gv0m59qr8wls7xs5yjx5llldyzack5kg5cg2mzk2nsml";
+  };
+
+  shellHook = ''
+    gksu () {
+      $@
+    }
+  '';
+
+  postPatch = ''
+    patchShebangs .
+  '';
+
+  buildInputs  = [
```

Comment:
According to https://github.com/KaizIqbal/Bibata_Cursor#dependencies
`inkscape` and `xcursorgen` should be in `nativeBuildInputs` (needed at build time only)

Upstream also mention a dependency on gnome-themes-standard (= `gnome-themes-extra` in nixpkgs)

Reply:
Resolved. I did read the docs, but I had no idea what the difference between the `*buildInput`'s were